### PR TITLE
PUBDEV-7070: Docs - AutoML Fixes

### PR DESCRIPTION
**Pull request by zainhaq-h2o**
Various fixes have been made to the AutoML docs.

See: https://0xdata.atlassian.net/browse/PUBDEV-7070

### CODE REVIEW

File: h2o-docs/src/product/automl.rst [hunk 1]
```diff
@@ -68,21 +68,18 @@ Optional Miscellaneous Parameters
 
 - `max_runtime_secs_per_model <data-science/algo-params/max_runtime_secs_per_model.html>`__: Specify the max amount of time dedicated to the training of each individual model in the AutoML run. Defaults to 0 (disabled). Note that setting this parameter can affect AutoML reproducibility.
 
--  `stopping_metric <algo-params/stopping_metric.html>`__: Specify the metric to use for early stopping.
-   The available options are:
+-  `stopping_metric <algo-params/stopping_metric.html>`__: Specify the metric to use for early stopping. Defaults to ``AUTO``. The available options are:
```
Comment (angela0xdata) [inline h2o-docs/src/product/automl.rst]: All automl parameter links need to include the data-science folder.
<data-science/algo-params/stopping_metric.html>

**Comment by angela0xdata** [comment]
@zainhaq-h2o only the stopping_metric description in AutoML needed to be reverted. Apologies if that was unclear in the jira ticket.

File: h2o-docs/src/product/automl.rst [hunk 2]
```diff
@@ -68,21 +68,18 @@ Optional Miscellaneous Parameters
...
+    - ``MSE``
+    - ``RMSE``
+    - ``MAE``
+    - ``RMSLE``
+    - ``AUC``
     - ``lift_top_group``
     - ``misclassification``
-    - ``aucpr``
     - ``mean_per_class_error``
     - ``custom`` (Python client only)
```
Comment (ledell) [inline h2o-docs/src/product/automl.rst]:
```suggestion
```

Comment (ledell) [inline h2o-docs/src/product/automl.rst]:
```suggestion
```

File: h2o-docs/src/product/automl.rst [hunk 3]
```diff
@@ -68,21 +68,18 @@ Optional Miscellaneous Parameters
...
     - ``custom_increasing`` (Python client only)
```
Comment (ledell) [inline h2o-docs/src/product/automl.rst]:
```suggestion
```

File: h2o-docs/src/product/automl.rst (hunk 2 above)
Comment (ledell) [reply h2o-docs/src/product/automl.rst]: We don't officially support this yet.

File: h2o-docs/src/product/automl.rst (hunk 3 above)
Comment (ledell) [reply h2o-docs/src/product/automl.rst]: We don't officially support this yet.

File: h2o-docs/src/product/automl.rst [hunk 4]
```diff
@@ -68,21 +68,18 @@ Optional Miscellaneous Parameters
...
 - `max_runtime_secs_per_model <data-science/algo-params/max_runtime_secs_per_model.html>`__: Specify the max amount of time dedicated to the training of each individual model in the AutoML run. Defaults to 0 (disabled). Note that setting this parameter can affect AutoML reproducibility.
 
--  `stopping_metric <algo-params/stopping_metric.html>`__: Specify the metric to use for early stopping.
-   The available options are:
+-  `stopping_metric <data-science/algo-params/stopping_metric.html>`__: Specify the metric to use for early stopping. Defaults to ``AUTO``. The available options are:
     
-    - ``auto``: This defaults to ``logloss`` for classification, ``deviance`` for regression, and ``anomaly_score`` for Isolation Forest. Note that custom and custom_increasing can only be used in GBM and DRF with the Python client. Must be one of: ``AUTO``, ``anomaly_score``. Defaults to ``AUTO``.
-    - ``anomaly_score`` (Isolation Forest only)
+    - ``AUTO``: This defaults to ``logloss`` for classification and ``deviance`` for regression. Note that ``custom`` and ``custom_increasing`` can only be used in GBM and DRF with the Python client.
     - ``deviance``
```
Comment (ledell) [inline h2o-docs/src/product/automl.rst]:
```suggestion
    - ``deviance`` (mean residual deviance)
```

File: h2o-docs/src/product/automl.rst [hunk 5]
```diff
@@ -68,24 +68,19 @@ Optional Miscellaneous Parameters
...
 - `max_runtime_secs_per_model <data-science/algo-params/max_runtime_secs_per_model.html>`__: Specify the max amount of time dedicated to the training of each individual model in the AutoML run. Defaults to 0 (disabled). Note that setting this parameter can affect AutoML reproducibility.
 
--  `stopping_metric <algo-params/stopping_metric.html>`__: Specify the metric to use for early stopping.
-   The available options are:
+-  `stopping_metric <data-science/algo-params/stopping_metric.html>`__: Specify the metric to use for early stopping. Defaults to ``AUTO``. The available options are:
     
-    - ``auto``: This defaults to ``logloss`` for classification, ``deviance`` for regression, and ``anomaly_score`` for Isolation Forest. Note that custom and custom_increasing can only be used in GBM and DRF with the Python client. Must be one of: ``AUTO``, ``anomaly_score``. Defaults to ``AUTO``.
-    - ``anomaly_score`` (Isolation Forest only)
-    - ``deviance``
+    - ``AUTO``: This defaults to ``logloss`` for classification and ``deviance`` for regression. Note that ``custom`` and ``custom_increasing`` can only be used in GBM and DRF with the Python client.
```
Comment (angela0xdata) [inline h2o-docs/src/product/automl.rst]: Remove this note regarding custom and custom_increasing

**Comment by angela0xdata** [comment]
also, the stopping_metric parameter topic (http://docs.h2o.ai/h2o/latest-stable/h2o-docs/data-science/algo-params/stopping_metric.html) needs to be updated to state that aucpr is not available for automl.

**Pull request merged**